netzwerk.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+---
+# Unser Netzwerk
+
```

Comment:
Vielleicht "Netzwerk von Initiativen, Vereinen und Verbänden"

Hier könnte noch ein Satz zur Erläuterung hin: hier tauchen keine Wirtschaftsunternehmen auf, sondern Initiativen, Vereine und Verbände in Hamburg oder mit einer Regionalgruppe in Hamburg, die sich - mindestens u.a. - für nachhaltige Wirtschaft einsetzen.

Und ich frage mich: sollen das alle sein, oder erstmal nur die, die auf dem Meetup waren? Je nachdem fehlen da welche.

Reply:
Wird erstmal zurück gestellt.